### PR TITLE
New comments

### PR DESCRIPTION
**Pull request by erikgarciam**
Hello I added some changes how comments are handled.
- DataServlet has been broken into multiple files in an attempt to be organized
- DataServletList was created to access datastore data and prepare it for script.js
- DataServletDelete is used by script.js when clicking delete button.
- Script.js create new elements in HTML for the index.html and bridges deletion between the servlet and html
- Style.css was modified to make comments look nicer rather than bullet points.
- Index.html was updated to have new form for displaying user choice number of comments along with option to delete.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServletList.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+            // Create class then convert to string. After converting
+            // to JSON string add to List to display later.
+            Comment commentcontainer = new Comment(name, lname, comment, id);
+            commentlist.add(commentcontainer);
+         }
+        }
+        // If user defines number iterate through for loop that many
+        // times to display user input value.
+        else{
```
Comment (avenkat-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServletList.java:75]: nit: space between else and {

File: portfolio/src/main/java/com/google/sps/servlets/DataServletList.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            // Retrieve contents from query/datastore
+            long id = entity.getKey().getId();
+            String name = (String) entity.getProperty("name");
+            String lname = (String) entity.getProperty("lname");
+            String comment = (String) entity.getProperty("comment");
+
+            // Create class then convert to string. After converting
+            // to JSON string add to List to display later.
+            Comment commentcontainer = new Comment(name, lname, comment, id);
+            commentlist.add(commentcontainer);
```
Comment (avenkat-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServletList.java:70]: You don't need to repeat this logic, as you can store the iterator in a variable. But also I recommend not allowing an unlimited number of comments to be displayed. Probably better to set a hard limit. Then maybe add pagination in the future.

Comment (erikgarciam) [reply portfolio/src/main/java/com/google/sps/servlets/DataServletList.java:70]: What data type would hold the variable?

Comment (erikgarciam) [reply portfolio/src/main/java/com/google/sps/servlets/DataServletList.java]: nvm figured it out.

File: portfolio/src/main/java/com/google/sps/servlets/DataServletList.java (hunk 1 above)
Comment (erikgarciam) [reply portfolio/src/main/java/com/google/sps/servlets/DataServletList.java]: Done!

**Pull request merged**